lib/semian.rb

```diff
@@ -275,7 +277,9 @@ def thread_safe=(thread_safe)
   private
 
   def create_circuit_breaker(name, **options)
-    circuit_breaker = options.fetch(:circuit_breaker, true)
+    not_disabled_by_env = !ENV.key?("SEMIAN_CIRCUIT_BREAKER_DISABLED")
+
+    circuit_breaker = options.fetch(:circuit_breaker, not_disabled_by_env)
```

Comment:
It's interesting that you chose this precedence. Could you expand on how you expect it to be used?

Reply:
Currently we have 2 different mechanisms of disabling circuit_breaker.
1. Resource configuration: `circuit_breaker: false` - It does not initialize circuit_breaker field in ProtectedResrouce.
2. With variable SEMIAN_CIRCUIT_BREAKER_DISABLED, that checked on each `acquire` and do `yield` if it is exists.

I merged both solutions in single one. (kinda get rid from second solution).

I have doubts to have ENV variable, because there is no documentation exists of using those variables. 
My first impression to implement the similar flow, 
but honestly I would just remove all usage of ENVs and
allow Apps decide create or not the Semian resources base on ENVs.

@byroot let me know what do you think.

Reply:
hmmmm good point